doc/source/user_guide/computation.rst

```diff
@@ -451,6 +451,10 @@ The list of recognized types are the `scipy.signal window functions
 * ``slepian`` (needs width)
 * ``exponential`` (needs tau).
 
+.. versionadded:: 1.2.0
+
+All Scipy window types, concurrent with your installed version, are recognized ``win_types``.
```

Comment:
which functions do we now support that we did not before? can u enumerate them

Reply:
Added. Eventually, we should remove all the enumerated supported window types as it will get stale with scipy developments

Reply:
maybe 
but we do parameter introspection now -
would be ok to fix this up i think and just point to the documentation of the actual window functions